buildsystem/codecompliance/authors.py

```diff
@@ -6,6 +7,45 @@
 
 import re
 
+def get_email_obfuscation_substitutions():
+    """
+    What tokens in email should be replaced with
+    """
+    return {
+    '.' : ' noot ',
+    '@' : ' à ',
+    'gmail' : '!gm!',
+    'com' : '!corn!',
```

Comment:
I think those rules might be too "advanced" already, as there's no obvious way how to encode one's email address.
Just imagine how a first-time contributor shouldn't fiddle around with adding the mail addr.